File: pinyougou-parent/pinyougou-content-interface/src/main/java/com/pinyougou/content/service/ContentCategoryService.java
package com.pinyougou.content.service;

import com.pinyougou.pojo.TbContentCategory;
import entity.PageResult;

import java.util.List;

public interface ContentCategoryService {

    List<TbContentCategory> findAll() throws Exception;

    PageResult<TbContentCategory> findByPage(TbContentCategory contentCategory, int pageNum, int pageSize) throws Exception;

    TbContentCategory findById(Long id) throws Exception;

    void save(TbContentCategory contentCategory) throws Exception;

    void delete(Long[] ids) throws Exception;
}

File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/plugins/angularjs/service/contentCategoryService.js
app.service('contentCategoryService', function ($http) {

    this.findByPage = function (page, size, searchEntity) {
        return $http.post('../contentCategory/findByPage.do?page=' + page + '&size=' + size, searchEntity);
    };

    this.findById = function (id) {
        return $http.get('../contentCategory/findById.do?id=' + id);
    };

    this.save = function (entity) {
        return $http.post('../contentCategory/save.do', entity);
    };

    this.del = function (ids) {
        return $http.get('../contentCategory/del.do?ids=' + ids);
    };

    this.findAll = function () {
        return $http.get('../contentCategory/findAll.do');
    };
});